### PR TITLE
tests: ping a fqdn instead of an ip when checking connectivity

### PR DESCRIPTION
**Pull request by lynxis**

### CODE REVIEW

**Comment by lynxis** [comment]
strange. it doesn't fix the problem. but it can definitly resolve it. Do they block ubuntu/debian repositorys?

**Comment by lynxis** [comment]
I still think it's a good idea, even it doesn't fix the whole problem. maybe travis is blocking access to the mirror server when executing the *script-section*

File: tests/tunneldigger.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def setup_template():
             raise RuntimeError("failed to start container")
 
     container.attach_wait(lxc.attach_run_command, ["dhclient", "eth0"])
-    check_ping(container, '8.8.8.8', 10)
+    check_ping(container, 'ccc.de', 10)
```
Comment (mitar) [inline tests/tunneldigger.py]: Could we ping instead `google-public-dns-a.google.com`? I believe it might have higher uptime than `ccc.de`. ;-)

**Comment by lynxis** [comment]
but if ccc.de is down, it's simpler to validate that, because 8.8.8.8 is an anycast address.

**Comment by mitar** [comment]
But we are not testing here if `ccc.de` is up or down. We need it to be up always, no? I think we should not have test fail and then have to go and see that it failed because `ccc.de` was down. I think it is better that test never fails for this reason. And using Google anycast will assure that.

**Comment by lynxis** [comment]
ok. i'll use the anycast address

**Comment by lynxis** [comment]
@mitar pong

**Comment by mitar** [comment]
Maybe we should then ping Travis CI servers? :-)

**Pull request merged**